boskos/Makefile

```diff
@@ -73,10 +74,16 @@ metrics-deployment:
 service:
 	kubectl apply -f service.yaml
 
-update-config: get-cluster-credentials
+activate-prow:
+	if [ -a $(SERVICE_ACCOUNT) ] ; \
```

Comment:
is it possible to put the `then` on the same line as the `if`?

Reply:
done